sample/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0-alpha02'
```

Comment:
you'll need to use the 1.0.0 version for us to merge this.  We can't publish a version of the tool with an alpha dependency.

Reply:
changed to 1.0.0